pinot-core/src/main/java/org/apache/pinot/core/operator/combine/BaseSingleBlockCombineOperator.java

```diff
@@ -111,7 +112,11 @@ protected void processSegments() {
   @Override
   protected void onProcessSegmentsException(Throwable t) {
     _processingException.compareAndSet(null, t);
-    _blockingQueue.offer(new ExceptionResultsBlock(t));
+    if (t instanceof BadQueryRequestException) {
```

Comment:
In ServerQueryExecutorV1Impl, we are setting `QUERY_VALIDATION_ERROR` for `BadQueryRequestException`. Do we require the same changes here? 

Either way I see this change only being made in BaseSingleBlockCombineOperator. What about  GroupByCombineOperator and BaseStreamingCombineOperator?

Reply:
> In ServerQueryExecutorV1Impl, we are setting QUERY_VALIDATION_ERROR for BadQueryRequestException. Do we require the same changes here?

It does. Without this change, generic `RuntimeException` is thrown to [QueryDispatcher.java](https://github.com/apache/pinot/pull/14917/files#diff-71eb98c311f40e7de5fa9750de41ff89c86aa9a9f7e485800992ff8f2e77b2c0) which will bypass that check.

Reply:
> Either way I see this change only being made in BaseSingleBlockCombineOperator. What about GroupByCombineOperator and BaseStreamingCombineOperator?

Yeah just noticed these implement the same interface. Will update them as well

Reply:
All patched.